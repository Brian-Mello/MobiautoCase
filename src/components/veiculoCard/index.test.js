import React from 'react';
import renderer from 'react-test-renderer';
import VeiculoCard from '.';

describe('testando integridade do VeiculoCard component', () => {
    it(' VeiculoCard snapshot', () => {
        const veiculoCard = renderer.create(<VeiculoCard/>).toJSON();
        expect(veiculoCard).toMatchSnapshot();
    });
}); 