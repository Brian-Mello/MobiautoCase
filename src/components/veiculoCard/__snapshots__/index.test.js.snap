// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testando integridade do VeiculoCard component  VeiculoCard snapshot 1`] = `
<div
  className="sc-AxjAm bVotXy"
>
  <header
    className="sc-AxirZ fgzlPM"
  >
    <b>
      Detalhes
    </b>
  </header>
  <main
    className="sc-AxiKw bAuRlc"
  >
    <div
      className="sc-AxhUy ghVtQG"
    >
      <div
        className="sc-AxgMl hWdEql"
      >
        <b>
          Marca:
        </b>
         
        <p
          className="sc-fznKkj fRNxDO"
        />
      </div>
      <div
        className="sc-Axmtr fjRRTW"
      >
        <b>
          Combutível:
        </b>
         
        <p
          className="sc-fznKkj fRNxDO"
        />
      </div>
    </div>
    <div
      className="sc-AxhUy ghVtQG"
    >
      <div
        className="sc-AxheI TibKV"
      >
        <b>
          Modelo:
        </b>
         
        <p
          className="sc-fznKkj fRNxDO"
        />
      </div>
      <div
        className="sc-fzozJi lbrYUd"
      >
        <b>
          Ano: 
        </b>
        <p
          className="sc-fznKkj fRNxDO"
        />
      </div>
    </div>
    <div
      className="sc-AxhUy ghVtQG"
    >
      <div
        className="sc-AxmLO dOvmXS"
      >
        <b>
          Sigla Combustível:
        </b>
         
        <p
          className="sc-fznKkj fRNxDO"
        />
      </div>
      <div
        className="sc-fzoLsD fEmEup"
      >
        <b>
          Código Fipe: 
        </b>
        <p
          className="sc-fznKkj fRNxDO"
        />
      </div>
    </div>
    <div
      className="sc-AxhUy ghVtQG"
    >
      <div
        className="sc-fzpans shZcn"
      >
        <b>
          Mês Referência: 
        </b>
        <p
          className="sc-fznKkj fRNxDO"
        />
      </div>
      <div
        className="sc-fzplWN bBpGtS"
      >
        <b>
          Tipo: 
        </b>
        <p
          className="sc-fznKkj fRNxDO"
        />
      </div>
    </div>
  </main>
  <footer
    className="sc-AxhCb fDdlYH"
  >
    <div
      className="sc-fznyAO jfbJtN"
    >
      <b>
        Valor: 
      </b>
      <p
        className="sc-fznKkj fRNxDO"
      />
    </div>
  </footer>
</div>
`;
